src/models/logbooks.js

```diff
@@ -0,0 +1,76 @@
+const mongoose = require('mongoose')
+ require('mongoose-double') (mongoose);
+
+ const SchemaTypes = mongoose.Schema.Types;
+
+
+const isrSchema= new mongoose.Schema({
+    ventas:{
+        type: SchemaTypes.Double,
```

Comment:
@AldaMayorgaDev Aquí entonces quedaria con type: Number
```suggestion
        type: Number,
```

y en los demás campos donde lo hayas utilizado vale